app/src/main/java/com/zeapo/pwdstore/crypto/PasswordCreationActivity.kt

```diff
@@ -223,9 +223,9 @@ class PasswordCreationActivity : BasePgpActivity(), OpenPgpServiceConnection.OnB
         val data = Intent()
         data.action = OpenPgpApi.ACTION_ENCRYPT
 
-        // EXTRA_KEY_IDS requires long[]
-        val longKeys = keyIDs.map { it.toLong() }
-        data.putExtra(OpenPgpApi.EXTRA_KEY_IDS, longKeys.toLongArray())
+        // pass enters the key ID into `.gpg-id`.
+        val keyIdFile = File(category.text.toString()).walkBottomUp().first { it.name == ".gpg-id" }
+        data.putExtra(OpenPgpApi.EXTRA_KEY_IDS, arrayOf(keyIdFile.readText()))
```

Comment:
We are creating an empty `.gpg-id` in `PasswordStore.createRepository`. If someone never clones a real pass repo, wouldn't this lead to problems?

Reply:
That's a good point which I certainly overlooked. I'm not sure of how I'd solve it tbh. Make users select a key during repo init and write that into `.gpg-id`?

Reply:
Yes, I think that would be the flow I would have expected as a user.